keg_elements/db/utils.py

```diff
@@ -178,18 +178,28 @@ def find_child(self, data):
     @no_autoflush
     def update(self):
         """Update the objects associated with the entity"""
```

Comment:
While we are in this code, can you give this a much better docstring describing what this function actually does.

Reply:
I updated the docstring of the `CollectionUpdater` class.